src/tram/tram/ml/base.py

```diff
@@ -1,26 +1,27 @@
-from abc import ABC, abstractmethod
-from datetime import datetime, timezone
-from io import BytesIO
-from os import path
 import pathlib
 import pickle
+import re
 import time
 import traceback
+from abc import ABC, abstractmethod
```

Comment:
Is there a reason why these imports were re-ordered? They were alpha sorted before, which is preferable.

Reply:
it's an isort configuration. each section is alphabetized, and it goes as follows: 

```
import python_builtins

import 3rd party (pypi modules)

import my library
```

and then each section is broken down into `import` statements, and then `from` statements

Reply:
I am in favor of what @m3mike suggests as the preferred approach to sorting imports. 
 - First built-ins
 - 3rd party
 - internal imports